crates/core/tedge_agent/src/file_transfer_server/error.rs

```diff
@@ -71,7 +71,7 @@ impl IntoResponse for FileTransferRequestError {
                 tracing::error!("{error_message}");
                 err.into_response()
             }
-            FromIo(_) | OtherDelete { .. } | OtherUpload { .. } => {
+            FromIo(_) | Delete { .. } | Upload { .. } => {
```

Comment:
note: Also not technically part of this PR, but [`use Enum::*` and matching variant names without `::` can be potentially a footgun if variants are removed](https://youtu.be/8j_FbjiowvE?si=VkIG0HQnCQwEd036&t=98).
Here I think it isn't that much of a problem, because all the variants have attached data, and if any binding here were to match "everything else" then matching individual fields would result in an compiler error, but still, I think doing `use FileTransferRequestError as E` and then matching on `E::...` would be preferable for new code.